### PR TITLE
13 feature about page

### PR DESCRIPTION
**Pull request by NDenheim**

### CODE REVIEW

File: src/pages/About/About.scss [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    -webkit-clip-path: ellipse(100% 100% at 50% 0%);
+    clip-path: ellipse(100% 100% at 50% 0%);
+  }
+
+  &__image--grey {
+    padding: 0;
+    transform: translate(0, -18vh);
+    margin: 0 calc((100vw - 350px) / 2);
+  }
+
```
Comment (olirob93) [inline src/pages/About/About.scss:61]: this modifier should be under the &__image bit

```

&__image {

 &--grey{
 padding: 0;
    transform: translate(0, -18vh);
    margin: 0 calc((100vw - 350px) / 2);
  }
}
}
```

File: src/pages/About/About.scss [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  &__subheading--white,
+  &__text--white {
+    color: $color-white;
+    transform: translate(0, -15vh);
+  }
+
+  &__subheading--white {
```
Comment (Charlie-robin) [inline src/pages/About/About.scss:68]: Can you put these modifiers in with the element?

```scss
  &__subheading {
    margin-bottom: 2vh;
    padding: 0 10vw;
    color: $color-charcoal;
    font-family: $modern-header;
    line-height: 1.2;
    &--white {
      color: $color-white;
    }
  }
```

File: src/pages/About/About.scss [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    text-align: right;
+  }
```
Comment (olirob93) [inline src/pages/About/About.scss:70]: Same thing as preveious comment

**Review by olirob93 — CHANGES_REQUESTED** [review]
2 small things on the scss other than that I think it's good

File: src/pages/About/About.tsx [hunk 4]
```diff
@@ -0,0 +1,149 @@
+import "./About.scss";
+import ballet from "../../assets/images/ballet.png";
+import reading from "../../assets/images/reading.png";
+import Header from "../../components/Header/Header";
+
+const About = () => {
+  return (
+    <div className="about">
+      <Header
```
Comment (Charlie-robin) [inline src/pages/About/About.tsx:9]: Can you combine this content with the <Layout/> component? At the moment you lose the navbar. Have a look at the Home component to see how it is implmented.

**Review by Charlie-robin — CHANGES_REQUESTED** [review]
Couple of things to tweak below.

**Review by olirob93 — APPROVED** [review]
looks good to me

**Review by Charlie-robin — APPROVED** [review]

**Pull request merged**